go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/jimmidyson/configmap-reload
 
-go 1.22
+go 1.24
```

Comment:
This doesn't bump go, this just bumps the language compatibility of the code which is actually still compatible with go 1.22. The version of go that released versions are built with is determined by the Docker image used to build it (just been updated to go1.24.5). For local development, the version of go in the toolchain directive will generally be used, go now downloads the toolchain version specified there and uses that to build the project, regardless of the installed version of go.